llmcam/store.py

```diff
@@ -61,6 +61,10 @@ def load_oas(
     overwrite=False
 )
 
+# %% ../nbs/10_store.ipynb 11
+from .fn_to_fc import complete, form_msgs, print_msgs
+from .oas_to_requests import generate_request
```

Comment:
https://chatgpt.com/share/673eba22-2014-8002-a316-c91383f783ca

should we move onto `from .fn_to_fc import *` ?